mozregression/cli.py

```diff
@@ -483,7 +485,12 @@ def validate(self):
                 options.launch = parse_date(options.launch)
                 self.action = "launch_nightlies"
             except DateFormatError:
-                self.action = "launch_inbound"
+                try:
+                    options.release_date = date_of_release(int(options.launch))
```

Comment:
So we discussed about that, this will fail with ValueError if we use a real changeset.